### PR TITLE
Abhishek - H2 Console

### LINKED ISSUE #11: FEATURE: H2 Console Enable
This issue stems from #6 and will allow the user to turn on/off the H2 console.

### PR DESCRIPTION
**Pull request by abhishekambastha04**
This closes #11
As a developer I can easily turn on the H2 option on the app navbar for any deployment through defining an environment variable H2 so that it is easy to access swagger when I need it, but I can easily turn it off when I don't. Allows developer to set a variable called H2 to true or false in their config variables. Adding these couple of lines on the application files allows for the H2 option on the navbar. Now, based on the dokku config value being true or false, it will give a H2 option.
Screen Shot 2024-05-22 at 2 16 58 AM
Above is when dokku config H2 is set to true.
Screen Shot 2024-05-22 at 2 23 36 AM
Above is when dokku config H2 is set to false.

### CODE REVIEW

File: src/main/resources/application-development.properties [hunk 1]
```diff
@@ -8,13 +8,16 @@ spring.datasource.url=jdbc:h2:file:./target/db-development
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
-spring.h2.console.enabled=true
-app.showSwaggerUILink=true
+#spring.h2.console.enabled=true
```
Comment (pconrad) [inline src/main/resources/application-development.properties]: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: src/main/resources/application-development.properties [hunk 2]
```diff
@@ -8,13 +8,16 @@ spring.datasource.url=jdbc:h2:file:./target/db-development
...
+spring.h2.console.enabled=${H2_CONSOLE_ENABLED:${env.H2_CONSOLE_ENABLED:true}}
+#app.showSwaggerUILink=true
```
Comment (pconrad) [inline src/main/resources/application-development.properties]: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: src/main/resources/application-production.properties [hunk 3]
```diff
@@ -3,6 +3,10 @@ spring.datasource.username=${JDBC_DATABASE_USERNAME}
 spring.datasource.password=${JDBC_DATABASE_PASSWORD}
 
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+# next line adds to UIlink
```
Comment (pconrad) [inline src/main/resources/application-production.properties]:
```suggestion
```

Comment seems superfluous here... this file is pretty self-explanatory.

File: src/main/resources/application-production.properties [hunk 4]
```diff
@@ -3,6 +3,10 @@ spring.datasource.username=${JDBC_DATABASE_USERNAME}
...
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+#next line adds the H2 console option
```
Comment (pconrad) [inline src/main/resources/application-production.properties]:
```suggestion
```
Comment seems superfluous here... this file is pretty self-explanatory.

**Review by pconrad — CHANGES_REQUESTED** [review]
Some commented out code and superflous comments to look at

**Review by NevMan1 — DISMISSED** [review]
LGTM

**Pull request merged**